codevalidator-service/pom.xml

```diff
@@ -185,5 +70,135 @@
 			<version>1.0-SNAPSHOT</version>
 		</dependency>
 	</dependencies>
+	
+	<profiles>
```

Comment:
- Always running an internal deployment step makes builds problematic for those downstream ...
- The Maven profile is activated by presence of the mandatory property, so existing automation/deployment on SITE's side won't have to do anything different.